tests/integration/cmd_push_test.go

```diff
@@ -330,8 +336,8 @@ var _ = Describe("odo push command tests", func() {
 				},
 			)
 
-			Expect(modifiedCatViewFile).NotTo(Equal(earlierCatViewFile))
-			Expect(modifiedCatServerFile).To(Equal(earlierCatServerFile))
+			Expect(modifiedCatPackageFile).To(Equal(earlierCatPackageFile))
```

Comment:
why is it other other way around - before it was `To` and now its `NotTo`?

Reply:
Because this time I am doing changes in server file, not in package.json